common/remote/rmtssh.cpp

```diff
@@ -447,7 +440,7 @@ class CFRunSSH: public CInterface, implements IFRunSSH
                             firsterr = false;
                             if (outbuf.length())
                                 outbuf.append('\n');
-                            outbuf.append("ERR: ");
```

Comment:
To be a bit clearer that this isn't an error per say, but the output of stderr from the remote connection.